doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_qsg.rst

```diff
@@ -75,22 +75,29 @@ For this quick start guide, we will install the following software:
 Building the SUIT sample
 ************************
 
-Start by building the SUIT sample:
+Start by building the SUIT smp_transfer sample:
 
 .. code-block:: console
 
-   west build -b nrf54h20dk/nrf54h20/cpuapp nrf/samples/suit/smp_transfer
+   west build --sysbuild -b nrf54h20dk/nrf54h20/cpuapp nrf/samples/suit/smp_transfer
 
-This command builds the SUIT sample for the nRF54H20 SoC.
+This command builds the SUIT smp_transfer sample for the nRF54H20 SoC.
```

Comment:
```suggestion
This command builds the :ref:`nrf54h_suit_sample` sample.
```
(The SoC is in the title of the sample doc.)

Reply:
Fixed